idp-binding-service/src/main/java/io/mosip/idp/binding/repository/PublicKeyRegistryRepository.java

```diff
@@ -31,6 +34,7 @@ public interface PublicKeyRegistryRepository extends JpaRepository<PublicKeyRegi
     Optional<PublicKeyRegistry> findByPublicKeyHashNotEqualToPsuToken(String publicKeyHash, String psuToken);
 
 	@Modifying
+	@Transactional
```

Comment:
Can we check because of this will we lose the overall transactional behaviour ?

Reply:
Transaction is moved to the service class.